common/strings.c

```diff
@@ -422,42 +422,24 @@ TDNFReplaceString(
     goto cleanup;
 }
 
-uint32_t
-TDNFTrimSuffix(
-    char* pszSource,
-    const char* pszSuffix
-    )
+uint32_t TDNFTrimSuffix(char *pszSource, const char *pszSuffix)
 {
-    uint32_t dwError = 0;
-    int nSourceStrLen = 0, nSuffixStrLen = 0;
+    char *ptr;
 
     if (IsNullOrEmptyString(pszSource) || IsNullOrEmptyString(pszSuffix))
     {
-        dwError = ERROR_TDNF_INVALID_PARAMETER;
-        BAIL_ON_TDNF_ERROR(dwError);
+        return ERROR_TDNF_INVALID_PARAMETER;
     }
 
-    nSourceStrLen = strlen(pszSource);
-    nSuffixStrLen = strlen(pszSuffix);
-
-    if (nSuffixStrLen > nSourceStrLen)
+    ptr = strcasestr(pszSource, pszSuffix);
```

Comment:
This ignore case. Is that intended? The original function is case sensitive.

Reply:
Will fix it. Thanks.